File: OOP-ClassesAndObjects_HristoChipev/Tire.cs
using System;
namespace OOP_Objects_Classes
{
    public class Tire
    {
        private double tirePressure;
        private int tireAge;

        public Tire(double tirePressure, int tireAge)
        {
            this.tirePressure = tirePressure;
            this.tireAge = tireAge;
        }

        public double TirePressure
        {
            get { return tirePressure; }
            set { tirePressure = value; }
        }

        public int TireAge
        {
            get { return tireAge; }
            set { tireAge = value; }
        }
    }
}

File: OOP_RobertProdanov/src/Tire.java
public class Tire {

    private int tireAge;
    private double tirePressure;

    public Tire(double tirePressure, int tireAge) {
        this.tireAge = tireAge;
        this.tirePressure = tirePressure;
    }

    public int getTireAge() {
        return tireAge;
    }

    public void setTireAge(int tireAge) {
        this.tireAge = tireAge;
    }

    public double getTirePressure() {
        return tirePressure;
    }

    public void setTirePressure(double tirePressure) {
        this.tirePressure = tirePressure;
    }

    @Override
    public String toString() {
        return "Tire{" +
                "tireAge=" + tireAge +
                ", tirePressure=" + tirePressure +
                '}';
    }
}
